microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -21,11 +21,11 @@ To get started using the Ad Selection API and test out the end-to-end flow, sign
 As a seller operating a supply-side platform or a buyer operating a demand-side platform, use this guide to:
 * Test the Ad Selection API on your site.
 * Sign up for the Ad Selection API limited preview.
-* Register for the Origin Trial.
-* Complete API attestation.
-* See the Ad Selection API and its usage, and deploy your services to Azure.
+* Register for the origin trial for the Ad Selection API.<!-- todo: merge w/ above -->
```

Comment:
list needs rewrite: 
* Merge dup steps, or nest/indent.
* Clarify "complete API attestation" - probably means Complete the API attestation by uploading the attesations JSON file.
@captainbrosset

Reply:
i plan to push commit to implement this, then close comment.